cli/tests/install.bats

```diff
@@ -479,3 +479,26 @@ EOF
   installed_flake=$(tomlq -r -c -t ".install.hello" "$MANIFEST_PATH")
   assert_equal "$installed_flake" "flake = \"$input_flake\""
 }
+
+# ---------------------------------------------------------------------------- #
+
+# bats test_tags=install:install-store-path
+@test "'flox install' install-store-path" {
+  "$FLOX_BIN" init
+  hello_store_path="$(nix build "github:nixos/nixpkgs/$PKGDB_NIXPKGS_REV_NEW#hello^out" --no-link --print-out-paths)"
+  vim_store_path_relative="$(nix build "github:nixos/nixpkgs/$PKGDB_NIXPKGS_REV_NEW#vim^out" --out-link ./result-vim --print-out-paths)"
```

Comment:
(nit) The out link / install path are relative but the store path isn't.
```suggestion
  vim_store_path="$(nix build "github:nixos/nixpkgs/$PKGDB_NIXPKGS_REV_NEW#vim^out" --out-link ./result-vim --print-out-paths)"
```

Reply:
yes that's by design
this is checking that i can reference a result link and it gets resolved to the store path it links to
i can break this into a separate test